app/client/components/user.tsx

```diff
@@ -73,26 +85,31 @@ const User = () => (
           </CancellationFlow>
         ))}
 
-      {Object.values(ProductTypes).map((productType: ProductType) => (
-        <PaymentUpdateFlow
-          key={productType.urlPart}
-          path={"/payment/" + productType.urlPart}
-          productType={productType}
-          currentStep={1}
-        >
-          <ConfirmPaymentUpdate
-            path="confirm"
+      {Object.values(ProductTypes)
+        .filter(
+          // i.e. don't create payment update flow for grouped product types
```

Comment:
This comment is helpful but perhaps having a shouldShowUpdatePaymentFlow boolean on the productType would make this even clearer?

Reply:
actually I didn't create an additional boolean for this because I want it to be based on whether the  product type is a grouping (i.e. it has grouped => specific function) - thereby minimising config.
However I did create a function in `productTypes.tsx` which removes the need for the comment 😄 
https://github.com/guardian/manage-frontend/blob/36305bb815ced4a071adb4a7486078e42edacb4d/app/shared/productTypes.tsx#L116-L117